portfolio/src/main/webapp/index.html

```diff
@@ -89,7 +89,19 @@ <h4>Join the Discussion!</h4>
           <input type="submit"/>
         </form>
       </div>
-      <h4 id="commentsCount">Loading...</h4>
+      <div id="commentsHeaderDisplay">
+        <span>
+          <h4>Displaying up to</h4>
+        </span>
+        <span>
+          <input type="number" id="numCommentsInput" name="numCommentsInput" min="0" max="1000" value="5"
+              onchange="updateNumCommentsDisplayed()">
+        </span>
+        <span>
+          <h4>comments:</h4>
```

Comment:
I haven't seen a live demo, but this seems a bit odd to me. Why is this a `<span>` and not a `<div>`? It should be on a new line, no?

Reply:
I used the <span> tags because I intended for that whole section, within that <div>, to be inline. I was thinking it would like:

Displaying up to [text-input] comments:

Should I still change something about it? I didn't know if using multiple span comments would be too redundant or not.